nidx/nidx_vector/src/data_point/mod.rs

```diff
@@ -81,18 +81,10 @@ where
     Dlog: DeleteLog,
 {
     let nodes_path = data_point_path.join(file_names::NODES);
-    let mut nodes_file_options = OpenOptions::new();
-    nodes_file_options.read(true);
-    nodes_file_options.write(true);
-    nodes_file_options.create(true);
-    let mut nodes_file = nodes_file_options.open(nodes_path)?;
+    let mut nodes_file = File::options().read(true).write(true).create(true).open(nodes_path)?;
 
     let hnsw_path = data_point_path.join(file_names::HNSW);
-    let mut hnsw_file_options = OpenOptions::new();
-    hnsw_file_options.read(true);
-    hnsw_file_options.write(true);
-    hnsw_file_options.create(true);
-    let mut hnsw_file = hnsw_file_options.open(hnsw_path)?;
+    let mut hnsw_file = File::options().read(true).write(true).create(true).open(hnsw_path)?;
```

Comment:
Nice! I didn't know this one and always wondered if there was something a bit more clear than OpenOptions. :heart: